src/server/oembed.js

```diff
@@ -12,9 +12,10 @@ const { URL } = require('url');
 
 const DEBUG = process.env.NODE_ENV !== 'production';
 
-router.get('/oembed', cors(), async (req, res, next) => {
+router.use(cors());
```

Comment:
just curious, does `cors()` and `cors({ origin: true })` makes any difference here?

Reply:
seems no difference here, do you prefer the latter one?

Reply:
former should be enough in this case